src/lib/completions.ts

```diff
@@ -56,18 +56,19 @@ export async function registerCompletions(ctx: ExtensionContext, core: Core): Pr
               return item;
             }): [];
 
-            const colorsCompletion = core.colorCompletions.map(({ label, detail, documentation}) => {
+            const colorsCompletion = core.colorCompletions.map(({ label, detail, documentation}, index) => {
               const color = new CompletionItem(label, CompletionItemKind.Color);
+              color.sortText = "-" + index.toString().replace(/\d+/g, ('00000000' + index).substr(-8));
```

Comment:
We can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart here

Reply:
will change 👍